### ISSUE TITLE
What tools should I use to label my own training set

### ISSUE THREAD

**Issue by slimsu** [navervision/mlsd#31]
Hello, I want to train my own data set, but I don't know which tool to use. It seems that labelimg and labelme are not suitable, so I would like to ask what tool should be used to mark the training set of pytorch version

**Comment by kachoali110** [comment]
@slimsu Hi, please did you find any suitable tool for labeling. I also trying to find but did not get any success.
